src/erl.rs

```diff
@@ -538,6 +538,12 @@ fn bare_clause_guard(guard: &TypedClauseGuard, env: &mut Env) -> Document {
 
         ClauseGuard::Float { value, .. } => value.to_string().to_doc(),
 
+        // TODO: .to_string() is not implemented?
+        ClauseGuard::Tuple { elems, ..} => elems
+            .map(|&x| x.to_string())
```

Comment:
We want to call the `bare_clause_guard` method on the element.

Reply:
The aim here is to generate the Erlang document `{Element1, Element2, Element3}` etc.

There's a function which will generate the right Erlang document when given an iterator of element documents -> https://github.com/gleam-lang/gleam/blob/0a89d55a8bb6fc5179efd9f592f14492a9775a28/src/erl.rs#L232-L237

There is an example of it being used here to make a pattern document: https://github.com/gleam-lang/gleam/blob/0a89d55a8bb6fc5179efd9f592f14492a9775a28/src/erl.rs#L322

The main difference is that rather than mapping over the elements with the `pattern` function we want to map over them with the `bare_clause_guard` function.